cli/pom.xml

```
@@ -105,6 +105,28 @@
           </archive>
         </configuration>
       </plugin>
+      <plugin>
+        <artifactId>maven-resources-plugin</artifactId>
+        <version>3.3.1</version>
+        <executions>
+          <execution>
+            <id>copy-resources</id>
+              <phase>validate</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${basedir}/target/package</outputDirectory>
```

```suggestion
              <outputDirectory>${project.build.directory}/package</outputDirectory>
```
This is more clean as we reconfigure `target` folder in a profile.
See https://cwiki.apache.org/confluence/display/MAVEN/Maven+Properties+Guide